tasks/services/works.js

```diff
@@ -222,14 +222,17 @@ function createCastOrderIntentTaskWorks (hook) {
       // TODO: IK: feels like a pretty sub-optimal way of querying, probably makes sense to have orderId as it's own column
       return taskPlans.find({
         query: {
-          taskRecipeId: 'castIntent',
-          params: {
-            $like: `%"orderId":${orderId}%`
-          }
+          taskRecipeId: 'castIntent'
+          // TODO: IK: can't compare JSON values in postgres, need another way to do it
+          // params: {
+          //   $like: `%"orderId":${orderId}%`
+          // }
         }
       })
         .then((taskPlans) => {
-          return Promise.all(taskPlans.map((taskPlan) => {
+          // TODO: IK: can't compare JSON values in postgres, need another way to do it
```

Comment:
https://www.postgresql.org/docs/current/static/functions-json.html

Reply:
@iainkirkpatrick i did a thing! [`queryByOrder`](https://github.com/root-systems/cobuy/blob/a38b8f61b7255024d95fc7c5e364696e21cefe3c/tasks/hooks/queryByOrder.js)

Reply:
oooh damn im intrigued to learn this wizardry @ahdinosaur 🎉 🎉 🎉